src/geefetch/data/get.py

```diff
@@ -89,7 +89,11 @@ def download_chip_ts(
     """Download a specific chip of data from the satellite."""
     bands = selected_bands if selected_bands is not None else satellite.default_selected_bands
     satellite.check_selected_bands(bands)
-    data = data_get_lazy(**data_get_kwargs)
+    try:
+        data = data_get_lazy(**data_get_kwargs)
+    except ValueError as e:
+        log.error(f"ValueError in data_get_lazy for {out}: {e}")
+        raise DownloadError from e
```

Comment:
The reason we are using DownloadError later is to be able to selectively retry on some exceptions (although retries are disabled for time series - I don't remember why). Is there a reason we want it here ?

Reply:
I see. I thought the errors would have been raised anyway - but indeed they were not because we did not call `future.result` during parallel downloads. Rather than logging within this function, which would only be catching `ValueError` during `Satellite.get`, I added a try/except `future.result()` call in `download` to log all exceptions.